assets/charts/components/azure-arc-onboarding/templates/remove.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    "helm.sh/hook": "pre-delete"
+  name: azure-arc-remove
+spec:
+  template:
+    spec:
+      serviceAccountName: azure-arc-onboarding
+      automountServiceAccountToken: true
```

Comment:
Do we need this both on SA and Pod spec?

Reply:
no. updated.